entity-services-examples/races-qc.xml

```diff
@@ -114,31 +113,32 @@ let $final-join :=
    =&gt;op:order-by(op:view-col("Race", "name"))
    
    
-return $final-join=&gt;op:result()</query><query name="Conversion" focus="false" listorder="10" taborder="13" active="true" database="5918579729198110779" server="10702847959065605031" database-name="Documents" server-name="App-Services" mode="xquery">xquery version "1.0-ml";
+return $final-join=&gt;op:result()</query><query name="Conversion" focus="true" listorder="13" taborder="13" active="true" database="12548380120383803910" server="14413040253700882037" database-name="entity-services-examples-content" server-name="entity-services-examples" mode="xquery">xquery version "1.0-ml";
 
-</query><query name="model-2-extract" focus="false" listorder="11" taborder="12" active="true" database="5918579729198110779" server="10702847959065605031" database-name="Documents" server-name="App-Services" mode="xquery">xquery version "1.0-ml";
+import module t = ""</query><query name="11. Extract 2" focus="false" listorder="12" taborder="12" active="true" database="12548380120383803910" server="14413040253700882037" database-name="entity-services-examples-content" server-name="entity-services-examples" mode="xquery">xquery version "1.0-ml";
 
 
 (: source loaded from csv :)
 
 import module namespace race2 = "http://grechaw.github.io/entity-types#Race-0.0.1-source-2" at "/ext/Race-0.0.1-source-2.xqy";
 
-let $doc := doc("2016-angel-island.csv-25.json" )
+declare variable $PATH as xs:string := "file:///MarkLogic/git";
+
+let $doc := doc($PATH || "/entity-services/entity-services-examples/data/third-party/csv/2016-angel-island.csv-25.json" )
 return
 (
 $doc,
 race2:extract-instance-Angel-Island( $doc )
-
 )
-</query><query name="01. load-report" focus="false" listorder="2" taborder="2" active="true" database="5918579729198110779" server="4141854128175404443" database-name="Documents" server-name="" mode="xquery">import module namespace es = "http://marklogic.com/entity-services" at "/MarkLogic/entity-services/entity-services.xqy"; 
+</query><query name="01. load-report" focus="false" listorder="2" taborder="2" active="true" database="12548380120383803910" server="14413040253700882037" database-name="entity-services-examples-content" server-name="entity-services-examples" mode="xquery">import module namespace es = "http://marklogic.com/entity-services" at "/MarkLogic/entity-services/entity-services.xqy"; 
 
 (
-"Raw docs: ", count(cts:search(collection("raw"), cts:true-query())),
-"Non raw: ", count(cts:search(collection(), cts:not-query(cts:collection-query("raw")))),
-"Race envelopes: ", count(//es:instance/Race),
-"Run envelopes: ", count(//es:instance/Run),
+"Raw docs (1354 expected): ", count(cts:search(collection("raw"), cts:true-query())),
+"Non raw (1310 expected): ", count(cts:search(collection(), cts:not-query(cts:collection-query("raw")))),
+"Race envelopes (4 expected): ", count(//es:instance/Race),
+"Run envelopes (1300 expected): ", count(//es:instance/Run),
```

Comment:
Nice addition of number of docs expected. I was wondering if after run all the docs intended were loaded.